input.go

```diff
@@ -724,6 +726,8 @@ type ComplianceFrameworksInput struct {
 
 	// The framework state to filter by. (Optional.)
 	State *ComplianceFrameworkState `json:"state,omitempty"`
+	// The framework states to filter by. (Optional.)
+	States *[]ComplianceFrameworkState `json:"states,omitempty"`
```

Comment:
do we need to also update a reference somewhere to use this? right now it looks like we're just updating the generated API code.

Reply:
Server changes are already implemented and shipped

Reply:
And need to get the library updated for the cnspec :)